Exiled.Events/EventArgs/Scp3114/StranglingFinishedEventArgs.cs

```diff
@@ -0,0 +1,68 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="StranglingFinishedEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Scp3114
+{
+    using API.Features;
+    using Interfaces;
+    using PlayerRoles.PlayableScps.Scp3114;
+
+    using Scp3114Role = Exiled.API.Features.Roles.Scp3114Role;
+
+    /// <summary>
+    ///     Contains all information after SCP-3114 finishes strangling a player (whether or not they kill the player).
+    /// </summary>
+    // ReSharper disable UnusedAutoPropertyAccessor.Global
+    // ReSharper disable MemberCanBePrivate.Global
+    public sealed class StranglingFinishedEventArgs : IScp3114Event
+    {
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="StranglingFinishedEventArgs" /> class.
+        /// </summary>
+        /// <param name="instance">
+        ///     The <see cref="Scp3114Strangle"/> instance which this is being instantiated from.
+        /// </param>
+        /// <param name="priorTarget">
+        ///     The prior <see cref="Scp3114Strangle.StrangleTarget"/> who was being targeted.
+        /// </param>
+        /// <param name="cooldown">
+        ///     The default cooldown that is used for the <see cref="Scp3114Strangle.Cooldown"/>.
+        /// </param>
+        public StranglingFinishedEventArgs(Scp3114Strangle instance, Scp3114Strangle.StrangleTarget priorTarget, float cooldown)
+        {
+            Player = Player.Get(instance.Owner);
+            Scp3114 = Player.Role.As<Scp3114Role>();
+            StrangleInfo = priorTarget;
+            PriorTarget = Player.Get(priorTarget.Target);
+            StrangleCooldown = cooldown;
+        }
+
+        /// <summary>
+        ///     Gets or sets the cooldown that will be triggered for the <see cref="Scp3114Strangle"/> instance.
+        /// </summary>
+        public float StrangleCooldown { get; set; }
```

Comment:
![image](https://github.com/Exiled-Team/EXILED/assets/66829532/666cfbaa-cd0a-48a0-b5e7-99994cd0ad55)

This should be double value

Reply:
![image](https://github.com/Exiled-Team/EXILED/assets/74625280/d2148918-c11e-4049-803a-1a91d2febfdc)

Reply:
Basegame is actually a float which is casted into a double

Reply:
I'll switch the event type but the parameter for the event constructor needs to stay a float or it may throw an exception